AndroidP/app/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 'android-P'
+    defaultConfig {
+        applicationId "com.novoda.androidp"
+        minSdkVersion 27
+        targetSdkVersion 28
```

Comment:
Yay the hack 💃 They should really document this :P Or maybe they don't want us to know...